src/printer.js

```diff
@@ -3005,17 +2931,6 @@ function printFunctionParams(path, print, options, expandArg, printTypeParams) {
     printed = path.map(print, paramsField);
   }
 
-  if (fun.defaults) {
```

Comment:
The coding style with single letter variables look definitely like the recast codebase.